[pcl_io] libusb is not set as a required dependency of opennni

<!--- WARNING: This is an issue tracker. Before opening a new issue make sure you read https://github.com/PointCloudLibrary/pcl/blob/master/CONTRIBUTING.md#using-the-issue-tracker. -->
**Describe the bug**

Hi, I'm vcpkg maintainer, after we updated pcl to 1.12.0, some users reported us that pcl failed to build using vcpkg (https://github.com/microsoft/vcpkg/issues/21241):
```
[462/463] : && /usr/bin/c++ -fPIC -fopenmp -g  people/CMakeFiles/pcl_ground_based_rgbd_people_detector.dir/apps/main_ground_based_people_detection.cpp.o -o bin/pcl_ground_based_rgbd_people_detector -L/mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib -Wl,-rpath,/mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libboost_filesystem.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libboost_date_time.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libboost_iostreams.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libboost_system.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libboost_regex.a  lib/libpcl_common.a  lib/libpcl_kdtree.a  lib/libpcl_search.a  lib/libpcl_features.a  lib/libpcl_sample_consensus.a  lib/libpcl_filters.a  lib/libpcl_io.a  lib/libpcl_visualization.a  lib/libpcl_segmentation.a  lib/libpcl_people.a  lib/libpcl_io.a  lib/libpcl_io_ply.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkIOGeometry-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libjsoncpp.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libpng16d.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libz.a  /usr/lib/libOpenNI.so  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkIOPLY-9.0.a  /usr/lib/x86_64-linux-gnu/libOpenGL.so  /usr/lib/x86_64-linux-gnu/libGLX.so  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkChartsCore-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkInfovisCore-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkInteractionImage-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkRenderingLOD-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkRenderingOpenGL2-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libGLEWd.a  /usr/lib/x86_64-linux-gnu/libGL.so  /usr/lib/x86_64-linux-gnu/libGLU.so  /usr/lib/x86_64-linux-gnu/libSM.so  /usr/lib/x86_64-linux-gnu/libICE.so  /usr/lib/x86_64-linux-gnu/libX11.so  /usr/lib/x86_64-linux-gnu/libXext.so  /usr/lib/x86_64-linux-gnu/libGLX.so  /usr/lib/x86_64-linux-gnu/libOpenGL.so  /usr/lib/x86_64-linux-gnu/libXcursor.so  /usr/lib/x86_64-linux-gnu/libXrender.so  /usr/lib/x86_64-linux-gnu/libXfixes.so  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkViewsContext2D-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkRenderingContext2D-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkViewsCore-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkInteractionWidgets-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkFiltersModeling-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkInteractionStyle-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkFiltersExtraction-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkFiltersStatistics-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkImagingFourier-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkParallelDIY-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkParallelCore-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkIOLegacy-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkIOXML-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkIOXMLParser-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkIOCore-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libdouble-conversion.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/liblzmad.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libexpat.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkRenderingAnnotation-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkRenderingFreeType-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libfreetyped.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libbz2d.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libbrotlidec-static.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libbrotlicommon-static.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libbrotlidec-static.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libbrotlicommon-static.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkFiltersHybrid-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkImagingColor-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkFiltersTexture-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkImagingGeneral-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkImagingSources-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkImagingHybrid-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkIOImage-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkDICOMParser-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkmetaio-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libpng16d.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libtiffd.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libjpeg.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/liblzmad.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libz.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkRenderingVolume-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkImagingCore-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkRenderingUI-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkRenderingCore-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkCommonColor-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkFiltersGeometry-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkFiltersSources-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkFiltersGeneral-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkFiltersCore-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkCommonComputationalGeometry-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkCommonExecutionModel-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkCommonDataModel-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libpugixml_d.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkCommonMisc-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkCommonSystem-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkCommonTransforms-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkCommonMath-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkCommonCore-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkloguru-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtksys-9.0.a  -ldl  /usr/lib/x86_64-linux-gnu/libX11.so  lib/libpcl_features.a  lib/libpcl_filters.a  lib/libpcl_search.a  lib/libpcl_kdtree.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libflann_cppd.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/liblz4d.a  lib/libpcl_sample_consensus.a  lib/libpcl_octree.a  lib/libpcl_ml.a  lib/libpcl_common.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libboost_filesystem.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libboost_date_time.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libboost_iostreams.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libboost_system.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libboost_regex.a  /usr/lib/gcc/x86_64-linux-gnu/11/libgomp.so  /usr/lib/x86_64-linux-gnu/libpthread.so  -lm && :
FAILED: bin/pcl_ground_based_rgbd_people_detector 
: && /usr/bin/c++ -fPIC -fopenmp -g  people/CMakeFiles/pcl_ground_based_rgbd_people_detector.dir/apps/main_ground_based_people_detection.cpp.o -o bin/pcl_ground_based_rgbd_people_detector -L/mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib -Wl,-rpath,/mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libboost_filesystem.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libboost_date_time.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libboost_iostreams.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libboost_system.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libboost_regex.a  lib/libpcl_common.a  lib/libpcl_kdtree.a  lib/libpcl_search.a  lib/libpcl_features.a  lib/libpcl_sample_consensus.a  lib/libpcl_filters.a  lib/libpcl_io.a  lib/libpcl_visualization.a  lib/libpcl_segmentation.a  lib/libpcl_people.a  lib/libpcl_io.a  lib/libpcl_io_ply.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkIOGeometry-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libjsoncpp.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libpng16d.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libz.a  /usr/lib/libOpenNI.so  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkIOPLY-9.0.a  /usr/lib/x86_64-linux-gnu/libOpenGL.so  /usr/lib/x86_64-linux-gnu/libGLX.so  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkChartsCore-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkInfovisCore-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkInteractionImage-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkRenderingLOD-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkRenderingOpenGL2-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libGLEWd.a  /usr/lib/x86_64-linux-gnu/libGL.so  /usr/lib/x86_64-linux-gnu/libGLU.so  /usr/lib/x86_64-linux-gnu/libSM.so  /usr/lib/x86_64-linux-gnu/libICE.so  /usr/lib/x86_64-linux-gnu/libX11.so  /usr/lib/x86_64-linux-gnu/libXext.so  /usr/lib/x86_64-linux-gnu/libGLX.so  /usr/lib/x86_64-linux-gnu/libOpenGL.so  /usr/lib/x86_64-linux-gnu/libXcursor.so  /usr/lib/x86_64-linux-gnu/libXrender.so  /usr/lib/x86_64-linux-gnu/libXfixes.so  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkViewsContext2D-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkRenderingContext2D-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkViewsCore-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkInteractionWidgets-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkFiltersModeling-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkInteractionStyle-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkFiltersExtraction-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkFiltersStatistics-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkImagingFourier-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkParallelDIY-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkParallelCore-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkIOLegacy-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkIOXML-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkIOXMLParser-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkIOCore-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libdouble-conversion.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/liblzmad.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libexpat.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkRenderingAnnotation-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkRenderingFreeType-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libfreetyped.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libbz2d.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libbrotlidec-static.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libbrotlicommon-static.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libbrotlidec-static.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libbrotlicommon-static.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkFiltersHybrid-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkImagingColor-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkFiltersTexture-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkImagingGeneral-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkImagingSources-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkImagingHybrid-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkIOImage-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkDICOMParser-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkmetaio-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libpng16d.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libtiffd.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libjpeg.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/liblzmad.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libz.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkRenderingVolume-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkImagingCore-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkRenderingUI-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkRenderingCore-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkCommonColor-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkFiltersGeometry-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkFiltersSources-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkFiltersGeneral-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkFiltersCore-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkCommonComputationalGeometry-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkCommonExecutionModel-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkCommonDataModel-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libpugixml_d.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkCommonMisc-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkCommonSystem-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkCommonTransforms-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkCommonMath-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkCommonCore-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtkloguru-9.0.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libvtksys-9.0.a  -ldl  /usr/lib/x86_64-linux-gnu/libX11.so  lib/libpcl_features.a  lib/libpcl_filters.a  lib/libpcl_search.a  lib/libpcl_kdtree.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libflann_cppd.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/liblz4d.a  lib/libpcl_sample_consensus.a  lib/libpcl_octree.a  lib/libpcl_ml.a  lib/libpcl_common.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libboost_filesystem.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libboost_date_time.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libboost_iostreams.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libboost_system.a  /mnt/d/Git/GitHub/robocon-test-code/build/vcpkg_installed/x64-linux/debug/lib/libboost_regex.a  /usr/lib/gcc/x86_64-linux-gnu/11/libgomp.so  /usr/lib/x86_64-linux-gnu/libpthread.so  -lm && :
/usr/bin/ld: lib/libpcl_io.a(openni_driver.cpp.o): undefined reference to symbol 'libusb_get_device_descriptor'
/usr/bin/ld: /lib/x86_64-linux-gnu/libusb-1.0.so.0: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
ninja: build stopped: subcommand failed.
```
And this error is for all the non-Windows platform.

**Context**

After read the related code, I found that if we need to use feature `opennni` or `opennni2` in non-Windows, `libusb` must be a required dependency of that feature. And the dependency relationship of openni/opennni2 does not match the cpp code.

**Expected behavior**

To use feature openni or openni2 in non-Windows, we must find libusb and expect to find it.

**Current Behavior**

Can't find the libusb functions.

**To Reproduce**

1. clone vcpkg.
2. run bootstrap script.
3. `.\vcpkg install pcl[openni]`

**Screenshots/Code snippets**

In order to help explain your problem, please consider adding
* screenshots of the GUI issues: N/A
* code snippets: 
1. https://github.com/PointCloudLibrary/pcl/blob/d1c69ee25b705952bd79510b91a1629c1035feeb/io/CMakeLists.txt#L112
2. https://github.com/PointCloudLibrary/pcl/blob/d1c69ee25b705952bd79510b91a1629c1035feeb/io/src/openni_camera/openni_driver.cpp#L58-L60
3. https://github.com/PointCloudLibrary/pcl/blob/d1c69ee25b705952bd79510b91a1629c1035feeb/io/src/openni_camera/openni_driver.cpp#L362
4. https://github.com/PointCloudLibrary/pcl/blob/d1c69ee25b705952bd79510b91a1629c1035feeb/cmake/Modules/FindOpenNI.cmake#L54-L56
5. https://github.com/PointCloudLibrary/pcl/blob/d1c69ee25b705952bd79510b91a1629c1035feeb/cmake/Modules/FindOpenNI2.cmake#L43-L46

**Your Environment (please complete the following information):**

 - OS: Ubuntu 20.04.3 LTS on Windows 10 x86_64 (WSL)
 - Compiler: g++ (Ubuntu 11.1.0-1ubuntu1~20.04) 11.1.0
 - PCL Version: 1.12.0
 - CMake version: 3.21.2

**Possible Solution**

Change `if(CMAKE_SYSTEM_NAME STREQUAL "Darwin")` to `if(NOT WIN32)` in the following code [4] and [5].

**Additional context**

N/A

--- a/cmake/Modules/FindOpenNI.cmake
+++ b/cmake/Modules/FindOpenNI.cmake
@@ -51,7 +51,7 @@ if(OPENNI_INCLUDE_DIR AND OPENNI_LIBRARY)
   mark_as_advanced(OPENNI_INCLUDE_DIRS)
 
   # Libraries
-  if(CMAKE_SYSTEM_NAME STREQUAL "Darwin")
+  if(NOT WIN32)
     find_package(libusb REQUIRED)
     set(OPENNI_LIBRARIES ${OPENNI_LIBRARY} libusb::libusb)
   else()

--- a/cmake/Modules/FindOpenNI2.cmake
+++ b/cmake/Modules/FindOpenNI2.cmake
@@ -41,7 +41,7 @@ if(OPENNI2_INCLUDE_DIR AND OPENNI2_LIBRARY)
   mark_as_advanced(OPENNI2_INCLUDE_DIRS)
 
   # Libraries
-  if(CMAKE_SYSTEM_NAME STREQUAL "Darwin")
+  if(NOT WIN32)
     find_package(libusb REQUIRED)
     set(OPENNI2_LIBRARIES ${OPENNI2_LIBRARY} libusb::libusb)
   else()

